packages/jest-util/src/convertDescriptorToString.ts

```diff
@@ -31,7 +29,9 @@ export default function convertDescriptorToString(
   const stringified = descriptor.toString();
   const typeDescriptorMatch = stringified.match(/class|function/);
   const indexOfNameSpace =
+    // @ts-ignore
     typeDescriptorMatch.index + typeDescriptorMatch[0].length;
+  // @ts-ignore
```

Comment:
would be cool to leave a comment why these are ignored

Reply:
they were actually ignored before, but flow stopped caring for some reason: https://github.com/facebook/jest/pull/7618/files#diff-71069ea0076ff927b4f346e20d962a5fL34

Reply:
Probably had more data and could infer this better. Not an issue though :)

Reply:
IDK. The one you commented on here is due to `stringified.search` not taking the second argument. The one above is since we don't check that `stringified.match` actually returns something. That should be fine, although it's super impressive if flow knows `/class|function/` will match always due to the `typeof descriptor !== 'function'` above

Reply:
pushed comments 🙂